packages/engine/src/HedronEngine/HedronEngine.ts

```diff
@@ -137,7 +137,11 @@ export class HedronEngine {
             debugScene.addPass(pass)
           })
         }
-        instance.update({ deltaFrame: 1, params: paramValues })
+        try {
```

Comment:
I think this fixed an issue with bad code in a sketch hard crashing the app, but I might be mistaken, would be good to test in isolation